client/stylesheets/_modal.scss

```diff
@@ -921,7 +925,87 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 	END END END END END END
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	Report Modal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+/*~~~~ Button ~~~~*/
+
+.modReportTrigger {
+	background-color: #fff !important;
+	color: #404040 !important;
+}
+
+.modReportTrigger:hover {
+	background-color: #d3d3d3 !important;
+}
 
+/*~~ Button End ~~*/
+
+/*~~~~ Listing ~~~~*/
+
+.modReportWarning {
+	margin-top: 60px !important;
+	margin-right: auto !important;
+	margin-left: auto !important;
+	width: 80%;
+}
+
+.modReportWarning img {
+	border-radius: 50%;
+}
+
+.modReportWarning li:first-child, .modReportWarning li:last-child {
```

Comment:
Selector `modReportWarning` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line